src/content/developers/tutorials/how-to-write-and-deploy-an-nft/index.md

```diff
@@ -1,18 +1,18 @@
 ---
 title: How to Write & Deploy an NFT (Part 1/3 of NFT Tutorial Series)
-description: This tutorial is Part 1 of a series on NFTs that will take you step by step on how to write and deploy a Non Fungible Token (ERC-721 token) smart contract using Ethereum and Inter Planetary File System (IPFS). In Part II, we will describe how to mint an NFT using our smart contract, and in Part III, we will explain how to view your freshly minted NFT on Metamask!
+description: This tutorial is Part 1 of a series on NFTs that will take you step by step on how to write and deploy a Non Fungible Token (ERC-721 token) smart contract using Ethereum and Inter Planetary File System (IPFS). In [Part II, we will describe how to mint an NFT using our smart contract](/developers/tutorials/how-to-mint-an-nft/), and in [Part III, we will explain how to view your freshly minted NFT on Metamask](/developers/tutorials/how-to-view-nft-in-metamask/)!
 author: "Sumi Mudgil"
-tags: ["NFTs", "ERC-721", "non fungible tokens", "Alchemy", "Solidity", "smart contracts"]
+tags: ["NFTs", "ERC-721", "Alchemy", "Solidity", "smart contracts"]
 skill: beginner
 lang: en
 sidebar: false
-published: 2020-04-22
+published: 2021-04-22
 ---
 With NFTs bringing blockchain into the public eye, now is an excellent opportunity to understand the hype yourself by publishing your own NFT (ERC-721 Token) on the Ethereum blockchain!
 
 Alchemy is extremely proud to be powering the biggest names in the NFT space, including Makersplace (recently set a record digital artwork sale at Christie’s for $69 Million), Dapper Labs (creators of NBA Top Shot & Crypto Kitties), OpenSea (the world’s largest NFT marketplace), Zora, Super Rare, NFTfi, Foundation, Enjin, Origin Protocol, Immutable, and more.
 
-In this tutorial, we will walk through creating and deploying an ERC-721 smart contract on the Ropsten test network using [Metamask](https://metamask.io/), [Solidity](https://docs.soliditylang.org/en/v0.8.0/), [Hardhat](https://hardhat.org/), [Pinata ](https://pinata.cloud/)and [Alchemy](https://dashboard.alchemyapi.io/signup?referral=affiliate:aab0c546-cc29-4180-8d31-fbce31367277) (don’t fret if you don’t understand what any of this means yet — we will explain it!).
+In this tutorial, we will walk through creating and deploying an ERC-721 smart contract on the Ropsten test network using [Metamask](https://metamask.io/), [Solidity](https://docs.soliditylang.org/en/v0.8.0/), [Hardhat](https://hardhat.org/), [Pinata](https://pinata.cloud/)and [Alchemy](https://dashboard.alchemyapi.io/signup?referral=affiliate:aab0c546-cc29-4180-8d31-fbce31367277) (don’t fret if you don’t understand what any of this means yet — we will explain it!).
```

Comment:
(Noted this on Part 3, but MetaMask uses two capitals, missed this before, sorry)

Reply:
and you removed the space that was in the wrong place, but we need it after the link. currently renders:
[Pinata](https://pinata.cloud/)and